packages/@atjson/document/src/index.ts

```diff
@@ -65,19 +65,48 @@ export {
   UnknownAnnotation
 };
 
+export function getConverterFor(from: typeof Document, to: typeof Document): never | ((doc: Document) => Document) {
+  let exports = (typeof window !== 'undefined' ? window : global) as any;
+
+  let converters = exports.__atjson_converters__;
+  let converter = converters ?
+    converters[from.contentType] ?
+      converters[from.contentType][to.contentType] :
+    null :
+  null;
+  if (converter == null) {
+    throw new Error(`🚨 There is no converter registered between ${from.name} and ${to.name}.\n\nRegister a converter for this:\n\n${from.name}.defineConverterTo(${to.name}, doc => {\n  // ❤️ Write your converter here!\n  return doc;\n});`);
+  }
+
+  return converter;
+}
+
 export default class Document {
   static contentType: string;
   static schema: Array<AnnotationConstructor<any, any>> = [];
+  static debug: boolean = false;
```

Comment:
✨

Reply:
I removed this since it wasn't being used :(